### ISSUE TITLE
SCSSLint::Linter::SpaceAfterPropertyColon raised unexpected error

### ISSUE THREAD

**Issue by philwhln** [brigade/scss-lint#803]
```
SCSSLint::Linter::SpaceAfterPropertyColon raised unexpected error linting file tmp/npm-10-dQy3FMn5/1451951520682-0.5848183375783265/package/test/fixtures/spec/spec/libsass-todo-tests/scss-tests/109_test_selector_interpolation_in_string/input.scss: 'undefined method `+' for nil:NilClass'
/Users/phil/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/scss_lint-0.48.0/lib/scss_lint/linter/space_after_property_colon.rb:80:in `whitespace_after_colon'
/Users/phil/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/scss_lint-0.48.0/lib/scss_lint/linter/space_after_property_colon.rb:16:in `visit_prop'
/Users/phil/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/sass-3.4.22/lib/sass/tree/visitors/base.rb:36:in `visit'
/Users/phil/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/scss_lint-0.48.0/lib/scss_lint/linter.rb:138:in `visit'
/Users/phil/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/scss_lint-0.48.0/lib/scss_lint/linter.rb:149:in `block in visit_children'
/Users/phil/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/scss_lint-0.48.0/lib/scss_lint/linter.rb:147:in `each'
/Users/phil/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/scss_lint-0.48.0/lib/scss_lint/linter.rb:147:in `visit_children'
/Users/phil/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/sass-3.4.22/lib/sass/tree/visitors/base.rb:36:in `block in visit'
/Users/phil/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/scss_lint-0.48.0/lib/scss_lint/linter/space_after_property_colon.rb:12:in `visit_rule'
/Users/phil/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/sass-3.4.22/lib/sass/tree/visitors/base.rb:36:in `visit'
/Users/phil/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/scss_lint-0.48.0/lib/scss_lint/linter.rb:138:in `visit'
/Users/phil/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/scss_lint-0.48.0/lib/scss_lint/linter.rb:149:in `block in visit_children'
/Users/phil/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/scss_lint-0.48.0/lib/scss_lint/linter.rb:147:in `each'
/Users/phil/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/scss_lint-0.48.0/lib/scss_lint/linter.rb:147:in `visit_children'
/Users/phil/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/sass-3.4.22/lib/sass/tree/visitors/base.rb:38:in `visit'
/Users/phil/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/scss_lint-0.48.0/lib/scss_lint/linter.rb:138:in `visit'
/Users/phil/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/scss_lint-0.48.0/lib/scss_lint/linter.rb:41:in `run'
/Users/phil/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/scss_lint-0.48.0/lib/scss_lint/runner.rb:53:in `run_linter'
/Users/phil/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/scss_lint-0.48.0/lib/scss_lint/runner.rb:35:in `block in find_lints'
/Users/phil/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/scss_lint-0.48.0/lib/scss_lint/runner.rb:33:in `each'
/Users/phil/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/scss_lint-0.48.0/lib/scss_lint/runner.rb:33:in `find_lints'
/Users/phil/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/scss_lint-0.48.0/lib/scss_lint/runner.rb:19:in `block in run'
/Users/phil/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/scss_lint-0.48.0/lib/scss_lint/runner.rb:18:in `each'
/Users/phil/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/scss_lint-0.48.0/lib/scss_lint/runner.rb:18:in `run'
/Users/phil/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/scss_lint-0.48.0/lib/scss_lint/cli.rb:74:in `scan_for_lints'
/Users/phil/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/scss_lint-0.48.0/lib/scss_lint/cli.rb:59:in `act_on_options'
/Users/phil/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/scss_lint-0.48.0/lib/scss_lint/cli.rb:33:in `run'
/Users/phil/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/scss_lint-0.48.0/bin/scss-lint:7:in `<top (required)>'
/Users/phil/.rbenv/versions/2.1.5/bin/scss-lint:23:in `load'
/Users/phil/.rbenv/versions/2.1.5/bin/scss-lint:23:in `<main>'
Report this bug at https://github.com/brigade/scss-lint/issues

To help fix this issue, please include:
- The above stack trace
- SCSS-Lint version: 0.48.0
- Sass version: 3.4.22
- Ruby version: 2.1.5
- Contents of /Users/phil/conf/scss-lint.yml
```

Here's the scss-lint.yml

```
# Default application configuration that all configurations inherit from.

scss_files: "**/*.scss"
plugin_directories: ['.scss-linters']

# List of gem names to load custom linters from (make sure they are already
# installed)
plugin_gems: []

linters:
  BangFormat:
    enabled: true
    space_before_bang: true
    space_after_bang: false

  BemDepth:
    enabled: false
    max_elements: 1

  BorderZero:
    enabled: true
    convention: zero # or `none`

  ColorKeyword:
    enabled: true

  ColorVariable:
    enabled: false

  Comment:
    enabled: true

  DebugStatement:
    enabled: true

  DeclarationOrder:
    enabled: true

  DisableLinterReason:
    enabled: false

  DuplicateProperty:
    enabled: true

  ElsePlacement:
    enabled: true
    style: same_line # or 'new_line'

  EmptyLineBetweenBlocks:
    enabled: false
    ignore_single_line_blocks: true

  EmptyRule:
    enabled: true

  ExtendDirective:
    enabled: false

  FinalNewline:
    enabled: false
    present: true

  HexLength:
    enabled: true
    style: short # or 'long'

  HexNotation:
    enabled: false
    style: lowercase # or 'uppercase'

  HexValidation:
    enabled: true

  IdSelector:
    enabled: true

  ImportantRule:
    enabled: true

  ImportPath:
    enabled: true
    leading_underscore: false
    filename_extension: false

  Indentation:
    enabled: false
    allow_non_nested_indentation: false
    character: space # or 'tab'
    width: 4

  LeadingZero:
    enabled: false
    style: exclude_zero # or 'include_zero'

  MergeableSelector:
    enabled: true
    force_nesting: true

  NameFormat:
    enabled: false
    allow_leading_underscore: true
    convention: hyphenated_lowercase # or 'camel_case', or 'snake_case', or a regex pattern

  NestingDepth:
    enabled: false
    max_depth: 3
    ignore_parent_selectors: false

  PlaceholderInExtend:
    enabled: true

  PropertyCount:
    enabled: false
    include_nested: false
    max_properties: 10

  PropertySortOrder:
    enabled: false
    ignore_unspecified: false
    min_properties: 2
    separate_groups: false

  PropertySpelling:
    enabled: true
    extra_properties: []

  PropertyUnits:
    enabled: true
    global: [
      'ch', 'em', 'ex', 'rem',                 # Font-relative lengths
      'cm', 'in', 'mm', 'pc', 'pt', 'px', 'q', # Absolute lengths
      'vh', 'vw', 'vmin', 'vmax',              # Viewport-percentage lengths
      'deg', 'grad', 'rad', 'turn',            # Angle
      'ms', 's',                               # Duration
      'Hz', 'kHz',                             # Frequency
      'dpi', 'dpcm', 'dppx',                   # Resolution
      '%']                                     # Other
    properties: {}

  QualifyingElement:
    enabled: false
    allow_element_with_attribute: false
    allow_element_with_class: false
    allow_element_with_id: false

  SelectorDepth:
    enabled: false
    max_depth: 3

  SelectorFormat:
    enabled: false
    convention: hyphenated_lowercase # or 'strict_BEM', or 'hyphenated_BEM', or 'snake_case', or 'camel_case', or a regex pattern

  Shorthand:
    enabled: true
    allowed_shorthands: [1, 2, 3]

  SingleLinePerProperty:
    enabled: true
    allow_single_line_rule_sets: true

  SingleLinePerSelector:
    enabled: false

  SpaceAfterComma:
    enabled: true

  SpaceAfterPropertyColon:
    enabled: true
    style: one_space # or 'no_space', or 'at_least_one_space', or 'aligned'

  SpaceAfterPropertyName:
    enabled: true

  SpaceAfterVariableName:
    enabled: true

  SpaceAroundOperator:
    enabled: true
    style: one_space # or 'no_space'

  SpaceBeforeBrace:
    enabled: true
    style: space # or 'new_line'
    allow_single_line_padding: false

  SpaceBetweenParens:
    enabled: true
    spaces: 0

  StringQuotes:
    enabled: false
    style: single_quotes # or double_quotes

  TrailingSemicolon:
    enabled: true

  TrailingWhitespace:
    enabled: true

  TrailingZero:
    enabled: true

  UnnecessaryMantissa:
    enabled: true

  UnnecessaryParentReference:
    enabled: true

  UrlFormat:
    enabled: true

  UrlQuotes:
    enabled: true

  VariableForProperty:
    enabled: false
    properties: []

  VendorPrefix:
    enabled: false
    identifier_list: base
    additional_identifiers: []
    excluded_identifiers: []

  ZeroUnit:
    enabled: true

  Compass::*:
    enabled: false```

**Comment by sds** [comment]
Hey @philwhln, thanks for the report.

Would really appreciate an SCSS code snippet that reproduces this problem (specifically, the contents of `package/test/fixtures/spec/spec/libsass-todo-tests/scss-tests/109_test_selector_interpolation_in_string/input.scss`). Without it, it's rather difficult to understand the situation that would lead to this. Thanks!

**Comment by philwhln** [comment]
Sorry, I no longer have more information on this and have forgotten what I was doing when I hit this. Feel free to close.
